src/Events.php

```diff
@@ -15,6 +15,7 @@
     const CITIZEN_INITIATIVE_CANCELLED = 'citizen_initiativecancelled';
     const CITIZEN_PROJECT_CREATED = 'citizen_project_created';
     const CITIZEN_PROJECT_UPDATED = 'citizen_project_updated';
+    const CITIZEN_PROJECT_APPROVE = 'citizen_project_approve';
```

Comment:
What about to rename this const in `CITIZEN_PROJECT_APPROVED` to keep the same events logic ?

Reply:
👍